scripts/generator/graphql_generator/csharp/SDK/CartLineItems.cs.erb

```diff
@@ -127,28 +130,38 @@ namespace <%= namespace %>.SDK {
         /// cart.LineItems.AddOrUpdate(variantId, 3);
         /// \endcode
         public void AddOrUpdate(string variantId, long? quantity = null, Dictionary<string, string> customAttributes = null) {
-            CartLineItem input = Get(variantId);
+            CartLineItem modifiedLineItem = Get(variantId);
+
+            if (modifiedLineItem != null) {
+                bool didModify = false;
 
-            if (input != null) {
                 if (quantity != null) {
-                    input.Quantity = (long) quantity;
+                    modifiedLineItem.Quantity = (long) quantity;
+                    didModify = true;
                 }
 
                 if (customAttributes != null) {
-                    input.CustomAttributes = customAttributes;
+                    modifiedLineItem.CustomAttributes = customAttributes;
+                    didModify = false;
```

Comment:
Could you quickly explain what custom attributes are?  Are line items not considered modified if we update custom attributes?